ui/components/src/LineChart/LineChart.tsx

```diff
@@ -177,11 +178,11 @@ export function LineChart({
       yAxis: getYAxes(yAxis, unit),
       animation: false,
       tooltip: {
-        show: showPointsOnHover,
+        show: showHoverEmphasis,
         trigger: 'axis',
         showContent: false,
         axisPointer: {
-          type: 'none',
+          type: showHoverEmphasis ? 'line' : 'none',
```

Comment:
Optional suggestion: I _think_ this bit of code might be a little easier to read if the variable was named after the status that informs these conditionals (e.g. `isOptimizedMode` or `isStandardMode` if you want the reverse) instead of a description of the outcome of that state (`showHoverEmphasis`).  For example:

```suggestion
          type: isOptimizedMode ? 'none' : 'line',
```

makes it a little clearer what's happening and why.

Reply:
Yeah I definitely like your suggested name better! I actually ended up removing these conditions from LineChart upon further testing ([see other comment](https://github.com/perses/perses/pull/821#discussion_r1036168943))